### DISCUSSION TITLE
Creating a new discussions opens a web browser

### DISCUSSION THREAD

**Discussion by Zerotask** [community/community#19478] [Mobile]
I'm using the android app with a tablet and I noticed from 2 previously created discussions that after the creation, a web browser opens it instead of opening it in the GH app

**Comment by candyho** [comment]
Hey, sorry for the hassle. Do you mind sharing which android device and app version you are using? That'd definitely help us to debug.

**Reply by Zerotask** [reply]
Samsung Galaxy Tab S5e with Android 11

**Reply by alcere** [reply]
Hi @Zerotask, thank you very much for reporting this issue!

I tried to reproduce it on my end without success. Could you confirm that this is still an issue? and if so, could you help us by sharing concrete steps to reproduce it? For instance, are you creating the Discussion from Home? from a repository? Is it only happening for a particular category of discussions?
Thanks!
